From 873ef6991aa7ea844c062ff083a852c1bdfba4c9 Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Mon, 1 Jul 2013 02:30:06 +0200
Subject: [PATCH 24/65] acerhk: change defaults

* enable polling, as usually required by dritek hardware.
* enable verbosity, this module is still under heavy development

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 drivers/platform/x86/acerhk.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/platform/x86/acerhk.c
+++ b/drivers/platform/x86/acerhk.c
@@ -78,11 +78,11 @@
 #define XSIZE     20
 
 /* Module parameters */
-static int poll = 1;
+static int poll = 0;
 static int autowlan;
 static int wlan_state = -1;
 static int bluetooth_state = -1;
-static int verbose;
+static int verbose=1;
 static unsigned int force_series;
 
 module_param(poll, int, 0444);
